tests/Localization/LtLtTest.php

```diff
@@ -17,210 +17,415 @@ class LtLtTest extends LocalizationTestCase
+        // 'Yesterday at 12:00 AM'
         'Yesterday at 00:00',
+
         // Carbon::parse('2018-01-04 00:00:00')->subDays(2)->calendar(Carbon::parse('2018-01-04 00:00:00'))
+        // 'Last Tuesday at 12:00 AM'
         'Last antradienis at 00:00',
+
         // Carbon::parse('2018-01-04 00:00:00')->subDays(3)->calendar(Carbon::parse('2018-01-04 00:00:00'))
+        // 'Last Monday at 12:00 AM'
         'Last pirmadienis at 00:00',
+
         // Carbon::parse('2018-01-04 00:00:00')->subDays(4)->calendar(Carbon::parse('2018-01-04 00:00:00'))
+        // 'Last Sunday at 12:00 AM'
         'Last sekmadienis at 00:00',
+
         // Carbon::parse('2018-01-04 00:00:00')->subDays(5)->calendar(Carbon::parse('2018-01-04 00:00:00'))
+        // 'Last Saturday at 12:00 AM'
         'Last šeštadienis at 00:00',
+
         // Carbon::parse('2018-01-04 00:00:00')->subDays(6)->calendar(Carbon::parse('2018-01-04 00:00:00'))
+        // 'Last Friday at 12:00 AM'
         'Last penktadienis at 00:00',
+
         // Carbon::parse('2018-01-03 00:00:00')->subDays(6)->calendar(Carbon::parse('2018-01-03 00:00:00'))
+        // 'Last Thursday at 12:00 AM'
         'Last ketvirtadienis at 00:00',
+
         // Carbon::parse('2018-01-02 00:00:00')->subDays(6)->calendar(Carbon::parse('2018-01-02 00:00:00'))
+        // 'Last Wednesday at 12:00 AM'
         'Last trečiadienis at 00:00',
+
         // Carbon::parse('2018-01-07 00:00:00')->subDays(2)->calendar(Carbon::parse('2018-01-07 00:00:00'))
+        // 'Last Friday at 12:00 AM'
         'Last penktadienis at 00:00',
+
         // Carbon::parse('2018-01-01 00:00:00')->isoFormat('Qo Mo Do Wo wo')
+        // '1st 1st 1st 1st 1st'
         '1 1 1 1 1',
+
         // Carbon::parse('2018-01-02 00:00:00')->isoFormat('Do wo')
+        // '2nd 1st'
         '2 1',
+
         // Carbon::parse('2018-01-03 00:00:00')->isoFormat('Do wo')
+        // '3rd 1st'
         '3 1',
+
         // Carbon::parse('2018-01-04 00:00:00')->isoFormat('Do wo')
+        // '4th 1st'
         '4 1',
+
         // Carbon::parse('2018-01-05 00:00:00')->isoFormat('Do wo')
+        // '5th 1st'
         '5 1',
+
         // Carbon::parse('2018-01-06 00:00:00')->isoFormat('Do wo')
+        // '6th 1st'
         '6 1',
+
         // Carbon::parse('2018-01-07 00:00:00')->isoFormat('Do wo')
+        // '7th 2nd'
         '7 1',
+
         // Carbon::parse('2018-01-11 00:00:00')->isoFormat('Do wo')
+        // '11th 2nd'
         '11 2',
+
         // Carbon::parse('2018-02-09 00:00:00')->isoFormat('DDDo')
+        // '40th'
         '40',
+
         // Carbon::parse('2018-02-10 00:00:00')->isoFormat('DDDo')
+        // '41st'
         '41',
+
         // Carbon::parse('2018-04-10 00:00:00')->isoFormat('DDDo')
+        // '100th'
         '100',
+
         // Carbon::parse('2018-02-10 00:00:00', 'Europe/Paris')->isoFormat('h:mm a z')
+        // '12:00 am cet'
         '12:00 priešpiet cet',
```

Comment:
not sure I get this

`priešpiet` literally means `before midday` 
12:00 is considered midday, so not sure why it's used here

Reply:
Nope `12:00 am` is `00:00` (midnight) So it seems correct.